command/meta.go

```diff
@@ -50,6 +50,11 @@ type Meta struct {
 	// Override certain behavior for tests within this package
 	testingOverrides *testingOverrides
 
+	// Override the set of provider plugin SHA256 digests we expect.
+	// If this is nil we will instead read from the provider lock file
+	// when setting up ContextOpts.
+	forceProviderSHA256s map[string][]byte
```

Comment:
gotcha. Didn't see an obvious test case, but it makes sense for pinning the versions from a plan.